app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe):
+    masks_to_buy = sum([
```

Comment:
Do not create a list to calculate a sum of generator elements
Use just `sum(1 for ....)`

Reply:
okay, thx